modules/account-lib/test/unit/coin/algo/transaction.ts

```diff
@@ -48,4 +51,128 @@ describe('Algo Transaction', () => {
       should.deepEqual(tx.canSign({ key: AlgoResources.accounts.account2.secretKey.toString('hex') }), true);
     });
   });
+
+  describe('Transaction type', () => {
+    it('field txType and tokenName are added on enableToken transaction', async () => {
```

Comment:
Please build the common part of tx in `beforeEach` so that there is less repetition of code.